src/app/shared/shared.module.ts

```diff
@@ -1,7 +1,7 @@
 import { NgModule } from "@angular/core";
 import { CommonModule } from "@angular/common";
 import { IonicModule } from "@ionic/angular";
-import { PLHMainHeaderComponent } from "./components/plh-main-header";
+import { PLHMainHeaderComponent } from "./components/header/header.component";
```

Comment:
nit(non-blocking): seeing as we've moved from `plh-main-header` maybe worth dropping the `plh` altogher and just using something like `AppHeaderComponent`
Also when referring to `MainHeader` is there some sort of alternative header? Not sure if we can drop the `Main`